sphinxcontrib/confluencebuilder/translator/wiki.py

```diff
@@ -1007,3 +1011,9 @@ def visit_raw(self, node):
         if 'confluence' in node.get('format', '').split():
             self.add_text(node.astext())
         raise nodes.SkipNode
+
+    def visit_todo_node(self, node):
```

Comment:
Since both `visit_todo_node` and `depart_todo_node` follow a non-wiki-specific pattern, these could instead be added into `translator/shared.py` instead. It may be nice to also include a comment that this is a `sphinx.ext.todo` hook instead of a standard hook.